File: Company/src/main/java/com/ftn/service/RobaUslugaService.java
package com.ftn.service;

import com.ftn.model.dto.RobaUslugaDTO;

import java.util.List;

/**
 * Created by JELENA on 22.6.2017.
 */
public interface RobaUslugaService {

    List<RobaUslugaDTO> read();

    RobaUslugaDTO create(RobaUslugaDTO robaUslugaDTO);


}

File: Company/src/main/webapp/service/robaUslugaService.js
/**
 * Created by JELENA on 23.6.2017.
 */

app.service('robaUslugaService', function($http){
    return {
        read: function(onSuccess, onError){
            $http.get('/api/robeUsluge').then(onSuccess, onError);
        }
    }
});
